### PR TITLE
Some upgrade fix

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 1]
```diff
@@ -31,9 +31,14 @@ Changes to apply
    using Sass in place of Less. The major change is that the variables will be prefixed by '$'
    instead of '@' and we will use rem as recommended unit.
 
-3. Docker version: To be able to use the mapfile faster we don't use the mako anymore for that,
-   to convert them use:
+3. Docker version: To be able to use the mapfile faster we don't use the mako anymore for that.
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]:
```suggestion
3. Docker version: To be able to deploy mapfile modifications faster we don't use the mako anymore for that.
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 2]
```diff
@@ -31,9 +31,14 @@ Changes to apply
...
+   1. To convert them use:
    `./docker-run make --always-make --makefile=CONST_convert2tmpl.mk to-tmpl`
+   If some files failes you can add some more variable in the file `vars_convert2tmpl.yaml`,
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]:
```suggestion
   If some files fail you can add more variable in the `vars_convert2tmpl.yaml` file,
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 3]
```diff
@@ -31,9 +31,14 @@ Changes to apply
...
+   and verify that the new variable is present in the config service as environment variable
+   (conserned files: `.env.mako` and `docker-compose.yaml`).
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]:
```suggestion
   (concerned files: `.env.mako` and `docker-compose.yaml`).
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 4]
```diff
@@ -31,9 +31,14 @@ Changes to apply
...
+   2. In the `vars` files you should also do the conversion present in the file `vars_convert2tmpl.yaml`
+   e.-g. `host: '{VISIBLE_WEB_HOST}'` meas that you shoud replace `{host}` by `{VISIBLE_WEB_HOST}`.
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]:
```suggestion
   e.-g. `host: '{VISIBLE_WEB_HOST}'` means that you shoud replace `{host}` by `{VISIBLE_WEB_HOST}`.
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 5]
```diff
@@ -31,9 +31,14 @@ Changes to apply
    using Sass in place of Less. The major change is that the variables will be prefixed by '$'
    instead of '@' and we will use rem as recommended unit.
 
-3. Docker version: To be able to use the mapfile faster we don't use the mako anymore for that,
-   to convert them use:
+3. Docker version: To be able to deploy mapfile modifications faster we don't use the mako anymore for that.
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: Suggestion:

Docker version: To be able to deploy mapfile modifications faster, we have switched from mako templating to a new templating mechanism.

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 6]
```diff
@@ -31,9 +31,14 @@ Changes to apply
...
+   1. To convert them use:
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: To convert your existing mako templates, use:

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 7]
```diff
@@ -31,9 +31,14 @@ Changes to apply
...
    `./docker-run make --always-make --makefile=CONST_convert2tmpl.mk to-tmpl`
+   If some files fail you can add more variable in the `vars_convert2tmpl.yaml` file,
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: If the conversion fails for some files, you can add more variables in the `vars_convert2tmpl.yaml` file,

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 8]
```diff
@@ -31,9 +31,14 @@ Changes to apply
...
+   and verify that the new variable is present in the config service as environment variable
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: and verify that the new variables are present in the config service as environment variables

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 9]
```diff
@@ -31,9 +31,14 @@ Changes to apply
...
+   (concerned files: `.env.mako` and `docker-compose.yaml`).
+   2. In the `vars` files you should also do the conversion present in the file `vars_convert2tmpl.yaml`
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: 2. You should adapt your `vars` files as illustrated in the file `vars_convert2tmpl.yaml`

**Comment by sbrunner** [comment]
Thanks @jwkaltz Done :-)

**Pull request merged**